html/browsers/history/the-history-interface/history_go_to_uri-1.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<script src="history.js"></script>
+<script>
+  onunload = function() {}
+
+  onload = function() {
+    if (!opener.started) {
+      queue_next();
+    } else {
+      opener.pages.push(id);
+      opener.start_test_wait();
+      if (!opener.gone) {
+          history.go("history_entry.html");
```

Comment:
yeah, that's what the test is testing: that .go() will fail in this case. IE11 supports it, but Edge does not (nor does Chrome, FF, etc), so we wanted to write a test to make sure we don't regress this interoperability.